packages/channel/src/channel.ts

```diff
@@ -222,9 +208,7 @@ class ChannelController<T> implements AsyncIterator<T> {
       this.buffer.add(value);
       return Promise.resolve();
     } else if (this.pushQueue.length >= MAX_QUEUE_LENGTH) {
-      throw new ChannelOverflowError(
```

Comment:
Renaming the class is an unnecessary breaking change, and I don’t prefer “CannotWriteToFullBufferError” over “ChannelOverflowError”